tests/features/content_version.feature

```diff
@@ -65,13 +65,77 @@ Feature: Content version
       | major | 1 |
       | minor | 0 |
       | patch | 0 |
+    # Set to draft by making a change.
+    When I click "Edit draft"
+    And I fill in "Title" with "Workflow node 3"
+    And I press "Save"
+    Then the node "Workflow node 3" should have the following version:
+      | major | 1 |
+      | minor | 1 |
+      | patch | 0 |
+    # Set back to validated.
+    When I select "Validated" from "Change to"
+    And I press "Apply"
+    And I wait for the batch to complete
+    Then the node "Workflow node 3" should have the following version:
+      | major | 2 |
+      | minor | 0 |
+      | patch | 0 |
+    # Set to draft by not making a change. It should increase the minor even
+    # if we didn't make any change
+    When I click "Edit draft"
+    And I press "Save"
+    Then the node "Workflow node 3" should have the following version:
+      | major | 2 |
+      | minor | 1 |
+      | patch | 0 |
+    # Set back to validated.
+    When I select "Validated" from "Change to"
+    And I press "Apply"
+    And I wait for the batch to complete
+    Then the node "Workflow node 3" should have the following version:
+      | major | 3 |
+      | minor | 0 |
+      | patch | 0 |
     # Set to Published and back to Draft.
     When I select "Published" from "Change to"
     And I press "Apply"
+    And I wait for the batch to complete
     And I click "New draft"
     And I press "Save"
-    Then the node "Workflow node 2" should have the following version:
-      | major | 1 |
+    Then the node "Workflow node 3" should have the following version:
+      | major | 3 |
+      | minor | 1 |
+      | patch | 0 |
+    # Set to published.
+    When I select "Published" from "Change to"
+    And I press "Apply"
+    And I wait for the batch to complete
+    Then the node "Workflow node 3" should have the following version:
+      | major | 4 |
+      | minor | 0 |
+      | patch | 0 |
+    # Unpublish the node by archiving it.
+    When I click "Unpublish"
+    And I select "Archived" from "Select the unpublishing state"
+    And I press "Unpublish"
+    Then the node "Workflow node 3" should have the following version:
```

Comment:
Before this we should make a new draft from the publish.

Reply:
Do you mean for coverage of the draft-after-publish scenario? I cover that here: https://github.com/openeuropa/oe_editorial/pull/115/files#diff-60d548e21baabe16fa3bf8d8d8c079fbae27ff0f68c5e52231667c4bd3827a54R169. There the test needed to be updated to assert that the unpublished revision that was created after a draft had been made, got its minor also increased.